rust/parser/test/mod.rs

```diff
@@ -617,12 +738,7 @@ get $x, $y;
 count;"#;
 
     let parsed = parse_query(query).unwrap().into_aggregate();
-    let expected = typeql_match!(rel("x").rel("y").isa("friendship"))
-        .get([
-            UnboundVariable::Concept(UnboundConceptVariable::named(String::from("x"))),
-            UnboundVariable::Concept(UnboundConceptVariable::named(String::from("y"))),
-        ])
-        .count();
+    let expected = typeql_match!(rel("x").rel("y").isa("friendship")).get([cvar("x").into(), cvar("y").into()]).count();
```

Comment:
There must be a way for us to write `.get([cvar("x"), cvar("y")])` instead of `.get([cvar("x").into(), cvar("y").into()])`

Reply:
Added `filter` macro after the in-person discussion. The reason: we can't have an array with mixed types.